server/hap2/test/TestHaplib.py

```diff
@@ -156,3 +159,18 @@ def test_create(self):
         self.assertEquals("", arm_info.last_failure_time)
         self.assertEquals(0, arm_info.num_success)
         self.assertEquals(0, arm_info.num_failure)
+
+class RabbitMQHapiConnector(unittest.TestCase):
+
+    def test_setup(self):
+        port = os.getenv("RABBITMQ_NODE_PORT")
+        amqp_address = os.getenv("RABBITMQ_NODE_ADDRESS")
+        transporter_args = {"direction": transporter.DIR_SEND,
+                            "amqp_broker": amqp_address,
+                            "amqp_port": port,
+                            "amqp_vhost": "test",
+                            "amqp_queue": "test_queue",
+                            "amqp_user": "test_user",
+                            "amqp_password": "test_password"}
```

Comment:
Now I have a question about treating "password".
Do these plugins receives passwords via only command line arguments?
If it's true, it's dangerous.
Any user can see it by `ps ax` command.